.gitlab-ci.yml

```
@@ -63,29 +64,20 @@ test:integration:
     - ./run.sh
 
 trigger:mender-dist-packages:
-  image: alpine
   stage: trigger
-  before_script:
-    - apk add --no-cache curl
-  script:
-    - curl -v -f -X POST
-      -F token=$MENDER_DIST_PACKAGES_TRIGGER_TOKEN
-      -F ref=master
-      -F variables[MENDER_CONFIGURE_VERSION]=$CI_COMMIT_REF_NAME
-      https://gitlab.com/api/v4/projects/14968223/trigger/pipeline
-  only:
-    - tags
-    - master
+  rules:
+    - if: $CI_COMMIT_TAG
+    - if: '$CI_COMMIT_BRANCH == "master"'
+  trigger:
+    project: Northern.tech/Mender/mender-dist-packages
+    branch: pr_104
```

Remember to switch before merge
```suggestion
    branch: master
```